src/components/catalog-cards.tsx

```diff
@@ -0,0 +1,24 @@
+import Card from './card';
+
+import { CamerasType } from '../types/cameras';
+
+type CatalogCardsProps = {
+  cameras: CamerasType;
```

Comment:
```suggestion
  cameras: Camera[];
```
Кто-то слабоумный придумал делать так как ты делаешь

Reply:
приставка Type неуместна. И так понятно что это тип, т.к. с большой буквы